Islandora-CLAW/islandora
# Media headers

**GitHub Issue**: 
- Resolves Islandora-CLAW/CLAW#721
- Part of Islandora-CLAW/CLAW#721
- Depends on https://github.com/Islandora-CLAW/islandora/pull/69

# What does this Pull Request do?

Adds two link headers to GET and HEAD responses for media entities:
- `<file_url>; rel="describes"; type="file_mimetype"` pointing to the where the media's file is on the filesystem
- '<rest_url>; rel="edit-media"; type="application/json"` pointing to the REST endpoint to issue PATCH requests to update the media's file

# What's new?

* Another ViewAlter as a service
* Functional tests

# How should this be tested?

Create a media, visit it, and then inspect the link headers sent in your GET response.  You should see two link headers, formatted as described above.

# Interested parties
@Islandora-CLAW/committers
